React/src/components/FormControl/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useForm } from 'react-hook-form';
+
+import { RegisterUser } from 'interfaces/RegisterUser';
+
+import styles from './styles.module.scss';
+
+export function FormControl() {
```

Comment:
FormControl isn't a good name, this is especific for the registration form.
```suggestion
export function RegisterForm() {
```